grommet-leaflet/src/layers/MarkerCluster/index.d.ts

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import * as L from 'leaflet';
+import 'leaflet.markercluster';
+
+export interface MarkerClusterProps {
+  icon?: () => any;
+  popup?: any;
```

Comment:
haven't confirmed this locally
```suggestion
  popup?: JSX.Element;
```

Reply:
Changed to `popup?: (any) => JSX.Element;`